src/model/wallet.js

```diff
@@ -47,7 +50,16 @@ export const walletInit = async (seedPhrase = null) => {
 };
 
 export const loadWallet = async () => {
-  const privateKey = await loadPrivateKey();
+  const savedVersion = loadKeyVersion();
```

Comment:
```suggestion
  const savedVersion = await loadKeyVersion();
```

Reply:
As discussed, you can ignore this comment since it's used everywhere in the file